### PR TITLE
finished settings configuration for CookiesStorage

### PR DESCRIPTION
**Pull request by pedrosilva0111**
Note: testing for auto domain is restricted because tests run on a single host, so a mock document object was created to simulate the behavior of setting cookies on a specific domain.
This option for testing also limits the variety of tests available, so there are only a few. Ideas for testing would be appreciated!

### CODE REVIEW

File: test/findAutoDomain_test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  });
+
+  it('returns null when domain is a public suffix', () => {
+    setupDocument('com');
+
+    expect(storage.findAutoDomain_(testDocument)).toBe(null);
+  });
+
+  it('returns null when domain does not contain a public suffix', () => {
+    setupDocument('[2001:0db8:85a3:0000:0000:8a2e:0370:7334]');
```
Comment (Alex7Li) [inline test/findAutoDomain_test.js:39]: I don't understand why you choose this value to pass into the code here, can you explain with a comment?

File: test/findAutoDomain_test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+goog.module('measurementLibrary.storage.CookiesStorage.testing.findAutoDomain_');
+goog.setTestOnly();
+
+const CookiesStorage = goog.require('measurementLibrary.storage.CookiesStorage');
+
+describe('the findAutoDomain method', () => {
```
Comment (Alex7Li) [inline test/findAutoDomain_test.js:6]: Capitalize the first letter of the outer describe to begin a sentence to be consistent within the library
'The \`findAutoDomain\` method of cookies storage'

File: src/storage/CookiesInterface.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
+goog.module('measurementLibrary.storage.CookiesStorage');
+
+/**
+ * A fancier storage that does cool things.
+ * @implements {StorageInterface}
+ */
```
[the remark this answers is not in the document]
Comment (pedrosilva0111) [reply src/storage/CookiesInterface.js:6]: Didn't catch this, thank you!

File: test/findAutoDomain_test.js (hunk 1 above)
Comment (pedrosilva0111) [reply test/findAutoDomain_test.js:43]: I left a comment explaining. Thanks!

File: src/storage/CookiesInterface.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+    if (document.cookie === oldCookies) {
+      return null;
+    }
+
+    return domain;
+  }
+
+  /** @override */
+  save(key, value, secondsToLive) {}
```
[the remark this answers is not in the document]
Comment (pedrosilva0111) [reply src/storage/CookiesInterface.js:80]: Yes! PR for those will be in soon after this one is merged

**Pull request merged**